libraries/AdaptiveExpressions/Expression.cs

```diff
@@ -734,7 +734,7 @@ public class FunctionTable : IDictionary<string, ExpressionEvaluator>
 #pragma warning restore CA1710 // Identifiers should have correct suffix
 #pragma warning restore CA1034 // Nested types should not be visible
         {
-            private readonly ConcurrentDictionary<string, ExpressionEvaluator> _customFunctions = new ConcurrentDictionary<string, ExpressionEvaluator>(StringComparer.InvariantCultureIgnoreCase);
```

Comment:
>StringComparer.InvariantCultureIgnoreCase [](start = 152, length = 41)

Did you want to drop the cases insensitivity?

Reply:
Yes. Personally think the custom functions should align with Builtin functions. (Case sensitivity).